docs/source/_src_user_guide/state_propagation/propagation_setup.rst

```diff
@@ -14,15 +14,18 @@ Propagation Setup
    propagation_setup/dynamics_types/mass
    propagation_setup/dynamics_types/multi_type
    propagation_setup/dynamics_types/multi_body
+   propagation_setup/dynamics_types/multi_arc
    propagation_setup/propagator_types
+   propagation_setup/integration_setup
    propagation_setup/termination_settings
    propagation_setup/dependent_variables
+   propagation_setup/console_output
    propagation_setup/propagation_architecture
    
 .. _propagation_setup_intro:
 
-Introduction
-============
+Dynamics types
```

Comment:
Now there is no text between Propagation Setup and Dynamics types. Maybe a small reminder of what propagation setup is about above Dynamics types could be useful.

Reply:
I've added (will push all changes in one go later):

In this part of the user guide, we will explain how to define settings for a numerical propagation of
different types of dynamics, with a wide variety of available settings. The propagator settings allow you to define:

- The exact state derivative model (e.g. the equations of motion that are to be solved), including initial and final conditions
- The numerical integrator that is to be used to solve these equations of motion
- The list of output variables that are to be provided after thee propagation is finished
- The output that is to be printed to the console during the propagation
- ...

Details on all of the above is discussed in the pages below. To define the propagation settings, 
you must have :ref:`created your physical environment <environment_setup>`.